app/views/directives/osc-image-summary.html

```diff
@@ -4,6 +4,12 @@
   <div ng-show="resource | annotation:'provider'">Provider: {{ resource | annotation:'provider' }}</div>
   <div ng-show="resource.metadata.namespace">Namespace: {{ resource.metadata.namespace }}</div>
 </div>
-<div class="resource-description gutter-bottom"
-     ng-bind-html="resource | description | linky">
+<div class="resource-description"
+     ng-bind-html="resource | description | linky"
+     ng-class="{'gutter-bottom': tag !== 'latest'}">
+</div>
+
+<div ng-if="tag === 'latest'" class="alert alert-info">
+  <span class="pficon pficon-info" aria-hidden="true"></span>
+  This builder image tracks the latest version of the {{name}} image, which could be updated to a newer major version in the future.  If your application will not run on a different major version go <a href="project/{{project}}/create" >back</a> and choose a specific version.
```

Comment:
so I believe @spadgett wrote a "back" attribute directive that you can put on a link and it will take you to the previous page in the history, then you wouldnt need to pass project into the osc-image-summary directive